Include/internal/pycore_pymem.h

```diff
@@ -95,6 +95,10 @@ struct _PyTraceMalloc_Config {
 PyAPI_DATA(struct _PyTraceMalloc_Config) _Py_tracemalloc_config;
 
 
+void *_PyObject_VirtualAlloc(size_t size);
+void _PyObject_VirtualFree(void *, size_t size);
```

Comment:
What do these do that the existing API doesn't?

Reply:
There is no existing API for virtual memory allocation.
We want to allocate virtual memory because we are implementing a stack.
We want big chunks of memory to avoid having to allocate new chunks too often, but we don't want to allocate physical memory for the unused top of each chunk.

Reply:
Looking at the implementation in obmalloc.c, the relationship to virtual memory is less than obvious.  We are making the same call that is used to allocate arenas, `_PyObject_Arena.alloc()`.  So from what you are saying, we are using virtual memory for our arenas as well.  Is that right?

Regardless, it would help to have a comment (probably in obmalloc.c) with some explanation on how these functions relate to virtual memory.